ivy/functional/backends/tensorflow/elementwise.py

```diff
@@ -51,7 +51,9 @@ def asin(x: Union[tf.Tensor, tf.Variable]) -> Union[tf.Tensor, tf.Variable]:
     return tf.asin(x)
 
 
-def asinh(x: Union[tf.Tensor, tf.Variable]) -> Union[tf.Tensor, tf.Variable]:
+def asinh(
+        x: Union[tf.Tensor, tf.Variable]
+) -> Union[tf.Tensor, tf.Variable]:
     return tf.asinh(x)
```

Comment:
cast x to float type before passing to tf.asinh function to prevent attribute type errors in tensorflow